### PR TITLE
Goa 2444

### PR DESCRIPTION
**Pull request by xwatkins**

### CODE REVIEW

File: app/advancedfilters/goTermsFilter.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
         </label>
       </li>
     </ul>
-    <div ng-show="selectedTermSize() > 20" class="text-center">
-      <strong>More than 20 terms selected.</strong>
+    <div ng-show="selectedTermSize() > 100" class="text-center">
```
Comment (ljgarcia) [inline app/advancedfilters/goTermsFilter.html]: I would suggest to move the limit to a scope variable.

Comment (xwatkins) [inline app/advancedfilters/goTermsFilter.html]: good idea, I'll use it as the upload limit too.

**Comment by ljgarcia** [comment]
Code looks fine but it does not seem to work for me. I go to "Explore Biology", try to add a predefined set "goslim_aspergillus", button says "Add 84 terms to selection", I click and nothing happens. And it is below the limit. Not sure what is going on.

**Comment by xwatkins** [comment]
This one is not on the slims page but in the GO term filter on the annotation page.

**Pull request merged**